### ISSUE TITLE
About the train data

### ISSUE THREAD

**Issue by szxSpark** [GeneZC/Chinese-NER#1]
Hi, How do you get the data about this task ,  MSRA (Bakeoff 2006). I can not google for that.

**Comment by GeneZC** [comment]
sorry for the late reply, I've put them in the /data directory of the repo.

**Comment by yqqqqqq** [comment]
> sorry for the late reply, I've put them in the /data directory of the repo.

Hi，I have been looking for MSRA dataset for several days,but I haven"t got it. Could you send me a copy? my email is 2431225222@qq.com   Thanks!

**Comment by GeneZC** [comment]
> sorry for the late reply, I've put them in the /data directory of the repo.
> 
> Hi，I have been looking for MSRA dataset for several days,but I haven"t got it. Could you send me a copy? my email is 2431225222@qq.com   Thanks!

sent

**Comment by light8lee** [comment]
Hi, @GeneZC .Are the `dev.txt`, `test.txt` and `train.txt` files in `data`  the original msra dataset?

**Comment by GeneZC** [comment]
@light8lee yes, they are. And char_*.txt are accordingly the data we used in our model.

**Comment by light8lee** [comment]
> @light8lee yes, they are. And char_*.txt are accordingly the data we used in our model.

Thank you.
